AAI/FAQ.md

```diff
@@ -4,21 +4,235 @@ title: AAI FAQ
+broker <- issuers : Return signed visa(s) for user
+note right
+[
+  "<visa1>",
+  "<visa2>"
+]
+end note
+end
+
+clearing <- broker : UserInfo Endpoint returns passport content (plain JSON, not signed JWT)
+note right
+{
+  "iss": "https://<issuer-website>/",
+  "sub": "<subject-identifier>",
+  "jti": "<token-identifier>",
+  "ga4gh_passport_v1": [
+    "<visa1>",
+    "<visa2>",
+   ...
+  ]
+}
+end note
+
+
+note over clearing, data  #FFCCCB
+Decision is made to release data using information contained in the
+passport - and this decision is coordinated with the data system to
+facilitate that actual data release.
+end note
+
+client <- clearing : Client is given data
+
+
+{% endplantuml %}
+
+<hr style="width: 10em; margin: 2em auto;"/>   
+
+### What is the complete end to end flow using token exchange?
+
+(last updated June 2022)
+
+The exchange flow does not ever distribute the initial *Passport-Scoped Access Token* beyond
+the client application. A token exchange operation is executed by the client, in
+exchange for a *Passport Token* that may be used downstream to access resources. In this example flow, the
+*Passport Token* is included as authorisation in the POST to a DRS server. The token
+exchange has also specified a known resource server website that will limit the audience
+of the *Passport Token*.
+
+{% plantuml %}
+
+hide footbox
+skinparam BoxPadding 10
+skinparam ParticipantPadding 20
+
+box "Researcher"  #eee
+actor       "User Agent"                as user
+participant Client                      as client
+end box
+
+box "AAI"
+participant Broker                      as broker
+collections "IdP"                       as idps
+end box
+
+box "Data Owner"
+collections "Visa Issuer(s)"            as issuers
+end box
+
+box "Data Holder"
+participant "Clearing House"            as clearing
+participant "Data"                      as data
+end box
+
+==OIDC==
+
+ref over user, client, broker, idps
+OIDC flow results in the client holding a *Passport-Scoped Access Token*.
+end ref
+
+==Exchange==
+
+client -> broker : Token exchange
+note right
+Content-Type: application/x-www-form-urlencoded
+
+grant_type=urn:ietf:params:oauth:grant-type:token-exchange
+requested_token_type=urn:ga4gh:params:oauth:token-type:passport
+subject_token=<Passport-Scoped Access Token>
+subject_token_type=urn:ietf:params:oauth:token-type:access_token
+audience=https://<drs-resource-server-website>/
+end note
+
+group#DeepSkyBlue #LightSkyBlue Informative Only (not defined in the specification)
+broker -> issuers : Fetch signed visa(s) for user
+broker <- issuers : Return signed visa(s) for user
+note right
+[
+  "<visa1>",
+  "<visa2>"
+]
+end note
+end
+
+client <- broker : Token exchange return
+note right
+{
+  "access_token": "<passport token (base64 encoded)>",
+  "issued_token_type": "<TBD>",
+  "token_type": "Bearer",
+  "expires_in": 60
+}
+
+▼ passport token content decoded from base64 (generally opaque to the client) ▼
+
+{
+  "typ": TBD,
```

Comment:
[Passport Header](https://ga4gh.github.io/data-security/1.2-draft-main/aai-openid-connect-profile#passport-header) section defines this.
```suggestion
  "typ": "vnd.ga4gh.passport+jwt",
```